src/app/admin/page/page.component.html

```diff
@@ -15,8 +15,8 @@
             <a href="#" >{{c.address}}</a>
           </mat-card-title>
           <mat-card-subtitle>
-            <span *ngIf="c.is_app !== 'true'">Site Page</span>
-            <span *ngIf="c.is_app === 'true'">App Page</span>
+            <span *ngIf="c.is_app !== true">Site Page</span>
```

Comment:
you can write
`c.is_app` instead of `c.is_app === true` and `!c.is_app` instead of `c.is_app !== true`